e2e/support/test_support.go

```diff
@@ -2685,7 +2685,44 @@ func CreateOperatorRoleBinding(t *testing.T, ctx context.Context, ns string) err
 	return nil
 }
 
-func CreateKamelPod(t *testing.T, ctx context.Context, ns string, name string, command ...string) error {
+func CreateKamelPodWithIntegrationSource(t *testing.T, ctx context.Context, ns string, name string, integration v1.ValueSource, command ...string) error {
```

Comment:
Let's take the opportunity and add a meaningful comment to let maintainers understand what this func is supposed to do.